### ISSUE TITLE
Support for WhisperX standalone

### ISSUE THREAD

**Issue by hlevring** [Purfview/whisper-standalone-win#174]
Would you consider to create a [whisperx](https://github.com/m-bain/whisperX) standalone CLI ? Whisperx also uses faster-whisper backend afaik, but provides far better word level timestamps.

**Comment by Purfview** [comment]
> ...but provides far better word level timestamps.

I think I didn't noticed better timestamps when I tested it. And I I've seen posts that WhisperX's  timestamps are worse.

Can you provide srt and json files made by WhisperX on this test file - https://we.tl/t-SHDM4cGKvB

**Comment by Purfview** [comment]
Looked at its Issues and at the top there is an issue with alignment -> https://github.com/m-bain/whisperX/issues/670

I've tested audio file from that post in the issue, when on some lines wav2vec alignment is a bit more precise, there are other lines where it's veeery inaccurate.
Here are srt files to compare:

[McDonalds_LNG_061019.WhisperX.zip](https://github.com/Purfview/whisper-standalone-win/files/14007973/McDonalds_LNG_061019.WhisperX.zip)
[McDonalds_LNG_061019.Standalone Faster-Whisper.zip](https://github.com/Purfview/whisper-standalone-win/files/14007975/McDonalds_LNG_061019.Standalone.Faster-Whisper.zip)

> Would you consider to create a [whisperx](https://github.com/m-bain/whisperX) standalone CLI ?

No. There are lots of stuff crammed in, many compatibility issues...  too much to bother for me.

Maybe I could incorporate wav2vec alignment alone from it, but if it has bigger issues than Whisper's original timestamps then maybe it's not worth it...

**Comment by hlevring** [comment]
Its a while back since I tested, so it would have made way more sense if I had opened the ticket with some proper documentation.

Something definitely goes wrong for whisperx with that file. I will have a closer look at the test file and run some more tests against whisperx next week when I have some time off. 

>Maybe I could incorporate wav2vec alignment alone from it, but if it has bigger issues than Whisper's original timestamps >then maybe it's not worth it...

Definitely agree. Going to make a little batch of test with both english and a couple of more languages.
